test/module/webpage/load-images-no.js

```diff
@@ -0,0 +1,52 @@
+//! phantomjs: --load-images=no
+
+// Compiled for debug, this tests for leaks when not loading images.
+//
+
+var url = TEST_HTTP_BASE + 'load-images.html';
+
+function do_test() {
+    var page = require('webpage').create();
+
+    var numImagesRequested = 0;
+    var numImagesLoaded = 0;
+    
+    page.onResourceRequested = this.step_func(function (resource) {
+        if (/\.png$/.test(resource.url)) ++numImagesRequested;
+    });
+
+    page.onResourceReceived = this.step_func(function (resource) {
+        if (resource.stage === 'end' && /\.png$/.test(resource.url)) ++numImagesLoaded;
+    });
+
+    function checkImageCounts ()
+    {
+        var imageCounts = page.evaluate (function() {    
+            "use strict";
```

Comment:
Oh, good point, it probably doesn't.  I think we serialize to text and recompile in the page context.  Leave that there and add one for the file, then?